app/app.scss

```diff
@@ -1,8 +1,16 @@
-﻿@import 'scss/core';
+﻿/*!
+ * NativeScript Theme v0.1.0 (https://nativescript.org)
+ * Copyright 2016-2016 The Theme Authors
+ * Copyright 2016-2016 Telerik
+ * Licensed under MIT (https://github.com/NativeScript/theme/blob/master/LICENSE)
+ */
+ 
+/**
+ * Base style (un-themed)
+ */
+@import 'scss/variables';
+@import 'scss/index';
 
-.main-page {
+// demo app
+@import 'demo';
```

Comment:
This is base default style. It imports the `demo` partial last to apply some basic styling to frame out the demo app which helps us develop these styles.